CompassOfWomanhood/translations/pl_PL/shithri_interjections.tra

```diff
@@ -10,6 +10,23 @@
 @1000009 = ~Jako żywo. Aż dziw bierze jak dobrze pasujo na glazyrki. Tylko małe nacioutko tu i tam...~
 @1000010 = ~Cha. Cha cha. Ostra z ciebie kobieta. Z tym. Ostrzem przy moim nadgarstku. A teraz pani wybaczy...~
 
+@1000100 = ~Ta, brzmi niegłupio. Pokażże tej ga... znaczy sie: tej kobicie, coś sie naumiał. */
```

Comment:
Why `*/` at the end of line? Should there be a comment?

Reply:
Looks like I didn't push the fix when I noted this bug too. ^^"

Pushed the fix.